### DISCUSSION TITLE
No trace in last version

### DISCUSSION THREAD

**Discussion by LaetitiaRedFroggy** [dadoonet/fscrawler#1071] [Q&A] [answered]
Hello,

I downloaded the last version (fscrawler-es7-2.7-20210104.212634-153.zip) in order to send documents in workplace search but when I launch the binary there is no trace even if i add --trace in the command.

Thanks in advance,

**Comment by dadoonet** [comment, marked as answer]
Could you share the log file?

**Reply by LaetitiaRedFroggy** [reply]
Thank you for your answer, here is the log file from command with --trace
[fscrawler.log](https://github.com/dadoonet/fscrawler/files/5816187/fscrawler.log)

**Reply by dadoonet** [reply]
I think this is the output in the console.
What about the logs in the `logs` dir?

**Reply by LaetitiaRedFroggy** [reply]
[fscrawler-2021-01-14-4.log](https://github.com/dadoonet/fscrawler/files/5816965/fscrawler-2021-01-14-4.log)
Indeed, it was console log, here is the log from logs dir.

Before this upgrade, I had logs in my console too.

**Reply by dadoonet** [reply]
Yeah. That's somehow a tradeoff. 
I'll try to see if we can have both activated.

**Reply by LaetitiaRedFroggy** [reply]
I was especially wondering if this was normal and if it was voluntary in this latest version? I was worried that it would come from my environment because I had a lot of problems to switch to java 15.

**Reply by dadoonet** [reply]
Was not you. Just a changed introduced by #1037

**Reply by LaetitiaRedFroggy** [reply]
Ok sorry I didn't see this feature ! Thank you very much for your reply and reactivity :-)
